### PR TITLE
Add the support blog post; minor tweaks to GGSC page; 

### PR DESCRIPTION
**Pull request by fubar2**
This is the first official announcement from the InterGalactic Ministry of Truth :)

The site tag does not seem to be used at the moment, so to tag my crazy deluge of posts, I just made that up. Are there plans to use those header tags?

### CODE REVIEW

**Comment by fubar2** [comment]
@bgruening: Now that they are blog posts, I deleted the working text from https://docs.google.com/document/d/1y7P7FKWRE3E4UnaXHLMm_f4tr5LFpRmtzXFvxqxdLCE/edit#heading=h.w8fh1ywdnhfw so it's a little less messy - can replace but seems sensible to only keep things we are working on there?

File: content/blog/2022-01-14-users-guide-to-contribution-ross/index.md [hunk 1]
```diff
@@ -1,12 +1,13 @@
 ---
 title: "A contributor's guide to the Galaxy"
-tease: "Contribution opportunities"
+tease: "Contribution opportunities for community members"
```
Comment (beatrizserrano) [inline content/blog/2022-01-14-users-guide-to-contribution-ross/index.md:3]: Do you think that this post is useful only if you're already a member? I'd say that some are good mechanisms to _become_ a member

File: content/blog/2022-01-14-users-guide-to-contribution-ross/index.md [hunk 2]
```diff
@@ -1,12 +1,13 @@
...
 authors: "Ross Lazarus"
 external_url: ""
 date: "2022-01-14"
 source_blog: ""
 source_blog_url: ""
 autotoc: false
+site: InterGalactic Ministry of Truth
```
Comment (beatrizserrano) [inline content/blog/2022-01-14-users-guide-to-contribution-ross/index.md:10]: 😆

File: content/blog/2022-01-16-open-scientific-computing-ross/index.md [hunk 3]
```diff
@@ -27,8 +34,10 @@ A common problem that prevents many researchers from routinely sharing their wor
...
+If you've managed to read this far, you may want to learn more about opportunities to help make open science computing more accessible. If so, you are cordially invited to [become involved in the Galaxy project](../2022-01-14-users-guide-to-contribution-ross) in any way you can.
```
Comment (beatrizserrano) [inline content/blog/2022-01-16-open-scientific-computing-ross/index.md:41]: Could this be a good place to link the contributing page too?

File: content/blog/2022-01-14-users-guide-to-contribution-ross/index.md (hunk 1 above)
Comment (fubar2) [reply content/blog/2022-01-14-users-guide-to-contribution-ross/index.md:3]: Yes, maybe best to revert to what it was - will do now

File: content/blog/2022-01-16-open-scientific-computing-ross/index.md (hunk 3 above)
Comment (fubar2) [reply content/blog/2022-01-16-open-scientific-computing-ross/index.md:41]: Maybe...it's a little awkward having two parallel resources :) 
Let me ponder it.

Comment (fubar2) [reply content/blog/2022-01-16-open-scientific-computing-ross/index.md:41]: @beatrizserrano: ok, reverted. I think we are done for now. The ministry for truth is established.

Comment (beatrizserrano) [reply content/blog/2022-01-16-open-scientific-computing-ross/index.md:41]: do we merge then?

Comment (fubar2) [reply content/blog/2022-01-16-open-scientific-computing-ross/index.md]: Both linked now - it's not pretty in some ways but they are different enough to be useful. The blog's not as exhaustive - maybe it needs merging but I'm over it for now.

File: content/blog/2022-01-18-reproducibility-and-support-bjoern/index.md [hunk 4]
```diff
@@ -0,0 +1,24 @@
+---
+site: freiburg
+tags: [comment]
+title: Galaxy as a sharing infrastructure and how this enables efficient user support
+authors: "Bjoern Gruening, Cristóbal Gallardo, Ross Lazarus"
```
Comment (bgruening) [inline content/blog/2022-01-18-reproducibility-and-support-bjoern/index.md:5]: What about Jen? I don't mind adding many authors that do support this blog.

File: content/blog/2022-01-16-open-scientific-computing-ross/index.md (hunk 3 above)
Comment (beatrizserrano) [reply content/blog/2022-01-16-open-scientific-computing-ross/index.md]: I think that they are addressing different audiences, depending on the reading preferences.

Comment (fubar2) [reply content/blog/2022-01-16-open-scientific-computing-ross/index.md]: @beatrizserrano: merge please. I'm done for now :)
Thanks!

File: content/blog/2022-01-18-reproducibility-and-support-bjoern/index.md (hunk 4 above)
Comment (fubar2) [reply content/blog/2022-01-18-reproducibility-and-support-bjoern/index.md:5]: @bgruening  - sure but who is Jen ? :) Need a name to fix it.

Comment (fubar2) [reply content/blog/2022-01-18-reproducibility-and-support-bjoern/index.md:5]: oh - Jen! Sure. I'll add her.

**Comment by fubar2** [comment]
@beatrizserrano 
Fixed a trivial hub landing page syntax error :)
Also fixed Jen's name to the full version on the support blog post - please merge?

**Review by beatrizserrano — APPROVED** [review]

**Comment by beatrizserrano** [comment]
thank you @fubar2!!

**Pull request merged**